### ISSUE TITLE
Urgent Table extraction 

### ISSUE THREAD

**Issue by PriyankaJain-1998** [DevashishPrasad/CascadeTabNet#139]
Hi, 
I got the result as it should be. 
But further I want to extract that particular region from an image, I search in "cascade_mask_rcnn_hrnetv2p_w32_20e.py" if anything related to contour is mentioned in that but didn't find anything. 
Please let me know how I can do that, Its urgent.

**Comment by seanbenhur** [comment]
Did you solved this?, I have tried numerous methods, but I can't find a single approach for extracting details from tables and saving them in a csv/json format except commercial OCR

**Comment by SAIVENKATARAJU** [comment]
HI @seanbenhur @PriyankaJain-1998 
 did you guys are able to figure out the approach?

**Comment by mohit-217** [comment]
Yes I solved the problem using open cv for text extraction in structured manner as oriented in different columns. i write the logic above OCR library

**Comment by PriyankaJain-1998** [comment]
@mohikumarr can you please provide the link to the solution

**Comment by mohit-217** [comment]
actually i solved using some logical approach ,what exactly you are facing
please elaborate more.

**Comment by mohit-217** [comment]
> Hi, I got the result as it should be. But further I want to extract that particular region from an image, I search in "cascade_mask_rcnn_hrnetv2p_w32_20e.py" if anything related to contour is mentioned in that but didn't find anything. Please let me know how I can do that, Its urgent.

ya sure,email your query @PriyankaJain-1998 freakintelligence@gmail.com

**Comment by jyotiyadav94** [comment]
@themohitkumar ,

Can you share the blog post you mentioned in the comment.

**Comment by vaibhavkansallumiq** [comment]
> Yes I solved the problem using open cv for text extraction in structured manner as oriented in different columns. i write the logic above OCR library

can you please share your logic with me also
